build.gradle

```diff
@@ -1,22 +1,25 @@
 buildscript {
+  ext.jdkVersion = 17
+  ext.javaClassVersion = 11
+
   ext.junitJupiterVersion = '5.6.1'
   // Releases: https://github.com/linkedin/rest.li/blob/master/CHANGELOG.md
-  ext.pegasusVersion = '29.46.8'
+  ext.pegasusVersion = '29.48.4'
   ext.mavenVersion = '3.6.3'
   ext.springVersion = '5.3.29'
   ext.springBootVersion = '2.7.14'
   ext.openTelemetryVersion = '1.18.0'
-  ext.neo4jVersion = '4.4.9'
-  ext.neo4jTestVersion = '4.4.25'
-  ext.neo4jApocVersion = '4.4.0.20:all'
+  ext.neo4jVersion = '5.14.0'
+  ext.neo4jTestVersion = '5.14.0'
+  ext.neo4jApocVersion = '5.14.0'
   ext.testContainersVersion = '1.17.4'
   ext.elasticsearchVersion = '2.9.0' // ES 7.10, Opensearch 1.x, 2.x
-  ext.jacksonVersion = '2.15.2'
+  ext.jacksonVersion = '2.15.3'
   ext.jettyVersion = '9.4.46.v20220331'
   ext.playVersion = '2.8.18'
   ext.log4jVersion = '2.19.0'
   ext.slf4jVersion = '1.7.36'
-  ext.logbackClassic = '1.2.13'
```

Comment:
Interesting that this dependency went backwards. 
from 1.2.13 to 1.2.12
Why?

Reply:
Nice catch, this seems like a bad merge maybe?

Reply:
definitely a mistake